docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -64,7 +71,40 @@ PIDFile=/run/user/1000/overlay-containers/de1e3223b1b888bc02d0962dd6cb5855eb0073
 WantedBy=multi-user.target default.target
 ```
 
-Create systemd unit files for a pod with two simple alpine containers. Note that these container services cannot be started or stopped individually via `systemctl`; they are managed by the pod service. You can still use `systemctl status` or journalctl to examine them.
+#### Generating systemd unit file for a container with `--new` flag
+
+ The `--new` flag basically generates systemd unit files that create and remove containers at service start and stop commands [Reffered by ExecStartPre and ExecStopPost service actions]. Such unit files are not tied to a single machine and can easily be shared and used on other machines.
+
+```
+$ sudo podman generate systemd --new --files --name bb310a0780ae
+# container-busy_moser.service
+# autogenerated by Podman 1.8.3
+# Fri Apr  3 09:40:47 EDT 2020
+
+[Unit]
+Description=Podman container-busy_moser.service
+Documentation=man:podman-generate-systemd(1)
+Wants=network.target
+After=network-online.target
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+ExecStartPre=/usr/bin/rm -f %t/%n-pid %t/%n-cid
+ExecStart=/usr/local/bin/podman run --conmon-pidfile %t/%n-pid --cidfile %t/%n-cid --cgroups=no-conmon -d -dit alpine
+ExecStop=/usr/local/bin/podman stop --ignore --cidfile %t/%n-cid -t 10
+ExecStopPost=/usr/local/bin/podman rm --ignore -f --cidfile %t/%n-cid
+PIDFile=%t/%n-pid
+KillMode=none
+Type=forking
+
+[Install]
+WantedBy=multi-user.target default.target
+```
+
+#### Generate systemd unit files for a pod with two simple alpine containers.
+
+ Note that these container services cannot be started or stopped individually via `systemctl`; they are managed by the pod service. You can still use `systemctl status` or journalctl to examine them.
```

Comment:
The services can be stopped (or) started via `systemctl` but they really should not. I believe to understand where you're pointing and suggest to instead explain that the dependencies inside the pod are reflected in the generated unit files: one unit for the pod (infra container) and one per container. As the pod is considered as a unit, `systemctl` should only be used on the pod unit but not on the container units.

Reply:
I have added more details referencing the above comment PTAL?

Also, This was an old reference I just refactored the heading is it out of context with the note tag added? Or is it good? 
Ref: https://github.com/containers/libpod/pull/5597/commits/e0131d908f954caa845d574b6b9d0e2361c3f4cd#diff-8faae638e5bcbc89b51451a155f1db92L67